instrumentation/opentelemetry-instrumentation-pika/src/opentelemetry/instrumentation/pika/pika_instrumentor.py

```diff
@@ -43,9 +54,7 @@ def _instrument_blocking_channel_consumers(
         consume_hook: utils.HookT = utils.dummy_callback,
     ) -> Any:
         for consumer_tag, consumer_info in channel._consumer_infos.items():
-            callback_attr = (
-                PikaInstrumentor._consumer_callback_attribute_name()
-            )
+            callback_attr = PikaInstrumentor.CONSUMER_CALLBACK_ATTR
             consumer_callback = getattr(consumer_info, callback_attr)
```

Comment:
Check if `consumer_info` has `callback_attr`. This will raise otherwise, which is not the wanted behavior.

Reply:
That's the current behavior - if `on_message_callback` isn't there, an exception would be thrown.
I don't mind simply skipping the instrumentation (skipping the `decorate_callback` part) in case that happens

Reply:
Well then, this should also be fixed now then. We don't want to raise. Your solution sounds great for this

Reply:
Done